src/search/engines/bingimages/dom.go

```diff
@@ -21,7 +21,8 @@ type bingImagesDomPaths struct {
 }
 
 var dompaths = bingImagesDomPaths{
-	Result: "ul.dgControl_list > li",
+	// for not() possible attributes are also: aria-live and vrhatt
+	Result: "ul.dgControl_list > li[data-idx] > div.iuscp:not([data-evt])",
```

Comment:
This breaks `make test-engines` bingimages. Reverting it seems to prove that everything else in this PR is good except this.